client/src/sections/Login/Login.css

```diff
@@ -2,16 +2,22 @@
     display: flex;
     flex-direction: column;
     align-items: center;
-    padding: 43rem;
+    padding-top: 25rem;
+    /* width: 5rem; */
 }
 
 .logo {
     margin: 2rem;
+    height: 8rem;
 }
 
 .login-heading {
     display: flex;
     flex-direction: column;
     align-items: center;
     padding: 2rem;
+}
+
+.Polaris-Card {
```

Comment:
I may have missed this earlier, but don't style on Polaris components directly. This will cause all Polaris Cards to be modified (check the Home page and you will see that all the cards there look weird now).

Reply:
how do i make the card wider then? removing the text made it too small

Reply:
Wrap a div around the ButtonGroup component and style that with padding.